test/integration/mergeLots.js

```diff
@@ -139,15 +139,15 @@ describe('Test merging lots', () => {
   // ===========================================================================
   // Verify we have created the lots, tags, etc
   it('Verify we created lot1 and its dupes (lot3, lot4, lot5)', () => {
-    return agent.get('/lot/dupes')
+    return agent.get('/lots_dupes')
```

Comment:
👀

Reply:
I was having problems making a unique url path for it because of all the optional arguments it supports.  I'm open to suggestions.